projects/core/components/textfield/textfield-auxiliary.ts

```diff
@@ -0,0 +1,21 @@
+import type {Provider, Signal, Type} from '@angular/core';
+import {computed, inject} from '@angular/core';
+import {tuiCreateToken, tuiProvide} from '@taiga-ui/cdk/utils/miscellaneous';
+
+import {TuiTextfieldComponent} from './textfield.component';
+
+export const TUI_AUXILIARY = tuiCreateToken(null);
+
+export function tuiAsAuxiliary(x: Type<unknown>): Provider {
+    return tuiProvide(TUI_AUXILIARY, x, true);
```

Comment:
I don't think it makes sense to provide it as multi token, we only need an array because there can be many of such content children, not because there can be multiple of auxiliaries on one injector.